modules/logging/src/test/kotlin/org/springframework/fu/module/logging/LogbackDslTest.kt

```diff
@@ -32,6 +32,9 @@ import java.io.File
 /**
  * @author Thomas Girard
  */
+
+private val TMP = System.getProperty("java.io.tmpdir").let(::File)
```

Comment:
I think I would prefer a `private val tmpdir: File` property in`LogbackDslTest` (test classes are instantiated just a single time).